Trimble.Modus.Components/Controls/Button/TMButton.cs

```diff
@@ -0,0 +1,407 @@
+
+        public static readonly BindableProperty BorderColorProperty =
+            BindableProperty.Create(nameof(BorderColor), typeof(Color), typeof(TMButton), (Color)BaseComponent.colorsDictionary()["TrimbleBlue"], propertyChanged: onBorderColorChanged);
+       
+        public static readonly BindableProperty TextColorProperty = 
+            BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(TMButton), defaultValue: Colors.White, propertyChanged: onTextColorChanged);
+       
+        public static readonly BindableProperty ButtonRadiusProperty =
+            BindableProperty.Create(nameof(Radius), typeof(int), typeof(TMButton), 4, propertyChanged: onButtonRadiusChanged);
+
+    
+        public bool IsFloatingButton
+        {
+            get { return (bool)GetValue(IsFloatingButtonProperty); }
+            set { SetValue(IsFloatingButtonProperty, value); }
+        }
+        public ButtonSize Size
+        {
+            get => (ButtonSize)GetValue(SizeProperty);
+            set => SetValue(SizeProperty, value);
+        }
+        public string Title
+        {
+            get => (string)GetValue(TitleProperty);
+            set => SetValue(TitleProperty, value);
+        }
+
+        public ImageSource IconSource
+        {
+            get => (ImageSource)GetValue(IconSourceProperty);
+            set => SetValue(IconSourceProperty, value);
+        }
+
+        public Command Command
+        {
+            get => (Command)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        public object CommandParameter
+        {
+            get => GetValue(CommandParameterProperty);
+            set => SetValue(CommandParameterProperty, value);
+        }
+
+        public Color TextColor
+        {
+            get { return (Color)GetValue(TextColorProperty); }
+            set { SetValue(TextColorProperty, value); }
+        }
+
+        public Color BorderColor
+        {
+            get { return (Color)GetValue(BorderColorProperty); }
+            set { SetValue(BorderColorProperty, value); }
+        }
+
+        public bool IsDisabled
+        {
+            get { return (bool)GetValue(IsDisabledProperty); }
+            set { SetValue(IsDisabledProperty, value); }
+        }
+
+
+        public new Color BackgroundColor
+        {
+            get { return (Color)GetValue(BackgroundColorProperty); }
+            set { SetValue(BackgroundColorProperty, value); }
+        }
+
+
+        public int Radius
+        {
+            get { return (int)GetValue(ButtonRadiusProperty); }
+            set { SetValue(ButtonRadiusProperty, value); }
+        }
+
+
+        public TMButton()
+        {
+
+            _titleLabel = new Label
+            {
+                TextColor = Colors.White,
+            };
+
+            _iconImage = new Image
+            {
+                WidthRequest = 24,
```

Comment:
@ElishaSamPeterPrabhu  This should be defined as a constant

Reply:
Defined as Constant